spark-schema-registry/src/main/scala/com/hortonworks/spark/registry/sparkavro/AvroDeserializer.scala

```diff
@@ -0,0 +1,420 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.hortonworks.spark.registry.sparkavro
```

Comment:
It looks like duplicated. Can we rename this package name?
```
com.hortonworks.spark.registry.sparkavro
com.hortonworks.spark.registry.avro
```

Reply:
Actually, this comes from Spark 2.4 `org/apache/spark/sql/avro/AvroDeserializer`, but the code is different. Do we need to re-sync as of today? Also, could you add the explanation in somewhere?